src/overlay_params.cpp

```diff
@@ -917,15 +919,17 @@ parse_overlay_config(struct overlay_params *params,
 
    //increase hud width if io read and write
    if (!params->width && !params->enabled[OVERLAY_PARAM_ENABLED_horizontal]) {
-      params->width = params->font_size * params->font_scale * params->table_columns * 4.6;
+      const auto real_size = params->font_size * params->font_scale;
+
+      params->width = static_cast<uint>(((1.0 + params->cellpadding_x * 2.0) * 4.6) * real_size * params->table_columns);
```

Comment:
mingw compiler doesn't recognize `uint`, casting as `unsigned int` should be more portable